modules/ROOT/nav.adoc

```diff
@@ -57,6 +57,7 @@
  ** xref:mule-debugger-view-reference.adoc[Mule Debugger View Reference]
  ** xref:to-start-server-debug-mode.adoc[Start the Mule Server In Debug Mode]
  ** xref:java-debugger-preference.adoc[Connect the Java Debugger]
+ ** xref:setting-inaccessible-fields.adoc[Set Inaccessible Fields]
```

Comment:
Why is this different than the H1?

Reply:
I changed the title in the doc and forgot to change it in the nav. Thanks!